podo/PodoCount_code/get_pod_props.py

```diff
@@ -12,6 +12,7 @@
 from skimage.color import rgb2gray

```

Comment:
we're not using this here and looks like you're importing everything from `sk` so we don't need to import `segmentation` again